warp-finn/src/main/java/com/schibsted/nmp/warp/brands/finn/FinnDarkColors.kt

```diff
@@ -197,6 +198,12 @@ internal object FinnComponentDarkColors : WarpComponentColors {
     override val navBar: WarpNavBarColors = FinnNavBarDarkColors
     override val tooltip: WarpTooltipColors = FinnTooltipDarkColors
     override val switch: WarpSwitchColors = FinnSwitchDarkColors
+    override val spinner: WarpSpinnerColors = FinnSpinnerDarkColors
+}
+
+private object FinnSpinnerDarkColors : WarpSpinnerColors {
```

Comment:
No need to create an object for these as the colors used are not custom component tokens, but already existing semantic color tokens. The spinner colors are defined in https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit?gid=1588196525#gid=1588196525